Add settings for Captcha support
--- a/mailman-suite_project/settings.py
+++ b/mailman-suite_project/settings.py
@@ -90,6 +90,7 @@
     #'allauth.socialaccount.providers.facebook',
     #'allauth.socialaccount.providers.twitter',
     #'allauth.socialaccount.providers.stackexchange',
+    #'captcha',
 )
 
 
@@ -479,6 +480,16 @@
 
 POSTORIUS_TEMPLATE_BASE_URL = 'http://localhost:8000'
 
+# # Uncomment the following lines for Captcha support
+# # Don't forget to add 'captcha' to INSTALLED_APPS.
+# ACCOUNT_SIGNUP_FORM_CLASS = 'django_mailman3.forms.CaptchaSignupForm'
+# import shutil
+# # Flite makes captchas audible for visually impaired users.
+# CAPTCHA_FLITE_PATH = shutil.which('flite')
+# # Sox adds random noise to prevent spammers calculating hashes of the
+# # captchas and creating rainbow tables of them.
+# CAPTCHA_SOX_PATH = shutil.which('sox')
+
 
 try:
     from settings_local import *
