From: Jonas Meurer <jonas@freesources.org>
Date: Fri, 14 Sep 2018 23:35:29 +0200
Subject: manage.py python3 interpreter

---
 mailman-suite_project/manage.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mailman-suite_project/manage.py b/mailman-suite_project/manage.py
index f9726f9..4eb6f34 100755
--- a/mailman-suite_project/manage.py
+++ b/mailman-suite_project/manage.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 import os
 import sys
 
